frontend/components/card/about/CardAboutOrganization.vue

```diff
@@ -20,14 +20,9 @@
         <h3 class="responsive-h3 text-left font-display">
           {{ $t("_global.about") }}
         </h3>
-        <IconEdit @click="openModal()" @keydown.enter="openModal()" />
-        <ModalEditAboutOrganization
```

Comment:
This right here ended up being why we were getting the extra modal closing on click, @mattburnett-repo :) `IconEdit` controls the modal at this point, so when we open it there were actually two modals on top of one another. We could tell this because the background of the `ModalEditAboutOrganization` was full white rather than `0.95`, or rather it was `0.95*0.95`.

Reply:
That makes sense. Thanks for puzzling this out!